lib/main_common.pm

```diff
@@ -1705,17 +1705,16 @@ sub load_extra_tests_sdk {
 
 sub load_extra_tests_docker {
     my ($image_names, $stable_names) = get_suse_container_urls();
-    return unless @$image_names;
 
     if (is_leap('15.1+') || is_tumbleweed) {
         loadtest 'containers/podman';
-        loadtest "containers/podman_image";
+        loadtest "containers/podman_image" if (@$image_names);
     }
 
     loadtest "containers/docker";
     loadtest "containers/docker_runc";
     loadtest "containers/container_base_images";
-    loadtest "containers/docker_image" if (is_sle(">=12-sp3") || is_opensuse);
+    loadtest "containers/docker_image" if (@$image_names && (is_sle(">=12-sp3") || is_opensuse));
```

Comment:
Sometimes, having a yaml schedule can save us from all these ugly conditions, just a thought :)

Reply:
True, but we need to modify some things first before we can switch to yaml. In particular we need to get rid of the `arg` variable in `main_common.pm`, that are needed to establish the tunnels ecc.

Reply:
Yes, this is for separate PR.